src/foam/parse/FScriptParser.java

```diff
@@ -203,6 +203,12 @@ public Object value() {
           public Object value() {
             return new RegExp();
           }
+        },
+        new AbstractLiteral("%")  {
```

Comment:
I'm open to other symbols for `IN` mlang, @kgrgreer what's your thought?

Reply:
MQL uses ':'. You should add to the JS FScriptParser as well.

Reply:
Updated.